purchases/src/main/kotlin/com/revenuecat/purchases/common/events/FeatureEvent.kt

```diff
@@ -0,0 +1,13 @@
+package com.revenuecat.purchases.common.events
+
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.customercenter.events.CustomerCenterEvent
+import com.revenuecat.purchases.paywalls.events.PaywallEvent
+
+interface FeatureEvent {
+    @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+    val paywallEvent: PaywallEvent? get() = null
+
+    @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+    val customerCenterEvent: CustomerCenterEvent? get() = null
+}
```

Comment:
This interface is a bit out of the norm. Maybe a sealed class fits here better.

Reply:
I wanted to do something similar to a protocol in swift 😅 
I guess that this could be seen like something like `AnyEvent` wrapper. I'll go with a sealed class and rename it